forge-gui/src/main/java/forge/player/HumanCostDecision.java

```diff
@@ -255,12 +255,19 @@ public PaymentDecision visit(final CostExile cost) {
             fromTopGrave = true;
         }
         boolean totalCMC = false;
+        boolean totalCMCgreater = false;
         String totalM = "";
         if (type.contains("+withTotalCMCEQ")) {
             totalCMC = true;
             totalM = type.split("withTotalCMCEQ")[1];
             type = TextUtil.fastReplace(type, TextUtil.concatNoSpace("+withTotalCMCEQ", totalM), "");
         }
+        if (type.contains("+withTotalCMCGE")) {
```

Comment:
isn't the same part needed there in CostExile?

https://github.com/Card-Forge/forge/blob/8fe5bb2086029b60a1cf7d4de40dbf33d7975f36/forge-game/src/main/java/forge/game/cost/CostExile.java#L193-L197